### DISCUSSION TITLE
Can it be used on centos7?

### DISCUSSION THREAD

**Discussion by skywander0** [facebook/CacheLib#155] [Q&A]
Can it be used on centos7?
When I use centos7, the following error will be reported
![image](https://user-images.githubusercontent.com/86784216/184191421-b2c8f621-4ce4-4e91-a86e-39bb6abe8fe6.png)

**Comment by skywander0** [comment]
If so, how to install it? Are there any related tutorials?

**Comment by agordon** [comment]
Hello,

Are you able to manually installed the packages listed in `https://github.com/facebook/CacheLib/blob/main/contrib/prerequisites-centos8.sh` ?
If so, that would go a long way towards building cachelib on CentOS 7.

**Reply by skywander0** [reply]
yes, I manually installed the libraries of the script, but I couldn't install "powertools", which made it impossible to execute；
When I execute 
"sudo dnf -y install ttps://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm",
 an error will be reported, and I will change the version to epel-release-latest -7.noarch.rpm without powertools
![image](https://user-images.githubusercontent.com/86784216/184276471-bd1c97a6-6870-459a-80de-2e4eb9491b6e.png)

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
